README.adoc

```diff
@@ -4,6 +4,13 @@
 :toc-title:
 :toclevels:
 
+[NOTE]
+====
+Each released `crc` binary expires 30 days after the release due to certificates embedded in the system bundle.
+Using an older `crc` binary will result in an expired certificates error.
+link:https://github.com/code-ready/crc/releases[Download the latest CodeReady Containers release] to resolve this error.
```

Comment:
system bundle -> virtual machine image/OpenShift cluster?
We should explain what to do after downloading the latest release (crc delete, which will lose all changes made to the cluster, crc setup, crc start).
The explanation here is much clearer than the one in proc_basic-troubleshooting.adoc so I'd copy it there too.